pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -157,7 +157,7 @@ def _find_names(self, lr_schedulers) -> List[str]:
         names = []
         for scheduler in lr_schedulers:
             sch = scheduler['scheduler']
-            if 'name' in scheduler:
+            if scheduler.get('name') is not None:
```

Comment:
this shall be the same, right?

Reply:
No because `'name' in {'name: None} == True` and we want to avoid that.

Reply:
yes, but below you adding it to the dictionary...

Reply:
`scheduler['name']` would work, but not `is in`